extensions/wikia/BannerNotifications/BannerNotificationsController.class.php

```diff
@@ -38,13 +38,22 @@ public static function addConfirmation( $message, $type = self::CONFIRMATION_CON
 				$_SESSION[self::SESSION_KEY] = [];
 			}
 
+			// Making sure if the message is not already set in the session.
+			// We're checking only the message text and not other message parameters,
+			// because the text is the most important here.
+			if ( in_array( $message, array_column( $_SESSION[self::SESSION_KEY], 'message' ) ) ) {
+				\Wikia\Logger\WikiaLogger::instance()
+					->debug( __METHOD__ . " - {$message}\n - already in the _SESSION array" );
```

Comment:
So there's no need to update this necessarily but FYI for future use; the `WikiaLogger` logging methods have a signature like:
```php
debug( $message, $params )
```
If you want to include interesting context around a debug call, its best to put it in the $param section so it can be queried easily from Kibana, e.g.:
```php
\Wikia\Logger\WikiaLogger::instance()->debug( "Message already in _SESSION array", [
    'method' => __METHOD__,
    'messageText' => $message,
    'issue' => 'SOC-1434', // I like adding this for filtering and context on the kibana side
] );
```